tests/test_rhalphalib.py

```diff
@@ -68,8 +68,7 @@ def test_rhalphabet(tmpdir):
         passCh = qcdmodel["ptbin%dpass" % ptbin]
         failObs = failCh.getObservation()
         qcdparams = np.array([rl.IndependentParameter("qcdparam_ptbin%d_msdbin%d" % (ptbin, i), 0) for i in range(msd.nbins)])
-        sigmascale = 10.0
-        scaledparams = failObs * (1 + sigmascale / np.maximum(1.0, np.sqrt(failObs))) ** qcdparams
+        scaledparams = failObs * (1 + 1.0 / np.maximum(1.0, np.sqrt(failObs))) ** qcdparams
```

Comment:
`** (qcdparams * sigmascale)`

Reply:
Actually, nevermind. I see what you're saying now.

Reply:
I put it back. The qcdparams end up having errors of ~0.1 as expected, but the values also decrease accordingly, so it does not affect the overall condition of the fit (at least in the test example).